includes/assets.php

```diff
@@ -262,6 +286,13 @@ public function register_scripts() {
 			 * @param   string $suffix The potential ".min" filename suffix.
 			 */
 			do_action( 'acf/register_scripts', $version, $suffix );
+
+			// Ensure WordPress scripts needed for command palette integration are available
+			if ( function_exists( 'wp_enqueue_script' ) ) {
+				wp_enqueue_script( 'wp-commands' );
+				wp_enqueue_script( 'wp-i18n' );
+				wp_enqueue_script( 'wp-dom-ready' );
+			}
```

Comment:
This shouldn't be necessary if they are listed as dependencies for other enqueued scripts.

Reply:
Fixed!